addons/sale_project/views/project_task_views.xml

```diff
@@ -118,29 +118,30 @@
                     </div>
                 </group>
             </xpath>
-            <xpath expr="//page[@name='settings']//field[@name='privacy_visibility']" position="before">
-                <field name="reinvoiced_sale_order_id" invisible="not allow_billable or not partner_id"/>
-                <label for="sale_line_id" invisible="not allow_billable or not partner_id"/>
-                <div 
-                    class="o_row" 
-                    invisible="not allow_billable or not partner_id">
-                    <field name="sale_line_id"
-                        groups="!sales_team.group_sale_salesman"
-                        options="{'no_create': True, 'no_edit': True, 'delete': False, 'no_open': True}"/>
-                    <field name="sale_line_id"
-                        groups="sales_team.group_sale_salesman"
-                        options="{'no_edit': True, 'delete': False}"
-                        context="{
-                            'form_view_ref': 'sale_project.sale_order_line_view_form_editable',
-                            'default_partner_id': partner_id,
-                            'default_company_id': company_id,
-                        }"/>
-                    <span
-                        class="fa fa-exclamation-triangle text-warning"
-                        title="The sales order associated with this project has been cancelled. We recommend either updating the sales order item or cancelling this project in alignment with the cancellation of the sales order."
-                        invisible="sale_order_state != 'cancel'"/>
-                </div>
-                <field name="sale_order_state" invisible="1"/>
+            <xpath expr="//page[@name='settings']//group//group" position="before">
```

Comment:
And instead, we should use the group name so that if any changes occur in the group, it won't affect here.